### ISSUE TITLE
How to commence with upgrade ?

### ISSUE THREAD

**Issue by jwcrawley** [dularion/streama#358]
Greetings. I saw your project when it was first mentioned on Hacker news. I started on the "streama-0.1.war" version. 

That version had to be started with a "grailsw run-war" and generated streama.mv.db  streama.trace.db in the relevant ~/streama/streama/target directory.

Now, I'm using streama-1.0.21.war and failing to determine how to migrate upwards while retaining the metadata and my files. 

I have over 300 GB of movies indexed by streama-0.1.war . I've tried everything... But nothing has succeeded thus far in allowing the newest version access to the metadata for all my added movies. The movie data still exists in /opt/movies/upload and Streama can still see it.

Suggestions on proper migration? Because I'm kind of flailing around with no good answers :/

**Comment by mongomay69** [comment]
i usually just copy my database folders over to where i have the new streama running and it works fine

**Comment by jwcrawley** [comment]
Well, I found out the problem. The version I was using required MySQL, and had no option for the localDB files in the H2 database format. 

What happened was that I ran the newest version in the old location's place, creating some DB files that really had nothing in them.

Where my data really was, was in my MySQL instance in streama DB. Once I downloaded the application.yml, and added MySQL creds, everything populated appropriately.

So, to fix:

1. Download the application.yml to the streama location.
2. comment out the local DB option and uncomment the MySQL option, add MySQL login/password
3. restart Streama
